src/legacy/core_plugins/data/public/search/search_bar/lib/saved_query_service.test.ts

```diff
@@ -140,6 +140,15 @@ describe('saved query service', () => {
       }
       expect(error).not.toBe(null);
     });
+    it("should throw an error if the saved query doesn't have a title", async () => {
```

Comment:
These should be single quotes instead of double quotes

Reply:
my linter was overzealous

Reply:
https://github.com/elastic/kibana/pull/45029/commits/071929576020a907b6547a68dc69c9713a564a69